testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -40,8 +40,9 @@ public enum WhitelistLogLines {
             // netty 4 which doesn't have the relevant native config in the lib. See https://github.com/netty/netty/pull/13596
             Pattern.compile(".*Warning: Please re-evaluate whether any experimental option is required, and either remove or unlock it\\..*"),
             Pattern.compile(".*Warning: The option '-H:ReflectionConfigurationResources=META-INF/native-image/io\\.netty/netty-transport/reflection-config\\.json' is experimental and must be enabled via.*"),
-            // TODO: remove next line when https://github.com/quarkusio/quarkus/issues/41351 gets fixed
+            // TODO: remove next lines when update to quarkus 3.13 or newer and fix from https://github.com/quarkusio/quarkus/issues/41351 is applied
```

Comment:
1. There is a PR with update to 3.13( https://github.com/quarkus-qe/quarkus-startstop/pull/381) which will hopefully merged this week. Do we need this patch anyway?
2. If yes, then please change to "remove next **two** lines", since this list has a tendency to grow :)

Reply:
1. If #381 is going to be merged soon, we can IMHO skip this one. But if it doesn't break break anything, I would merge this one just to be sure.
2. Done